.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish a release
+
+on:
+  push:
+    tags:
+      - 'v**'
```

Comment:
not really necessary, unless you plan to have different tags not linked to a version.

Reply:
It is no longer worth